### PR TITLE
[RHPAM-4740] - Unable to connect to external PostgreSQL DB over SSL f…

### PR DESCRIPTION
**Pull request by spolti**
…rom kie-server on OCP

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: jboss-kie-db-drivers/configure.sh [hunk 1]
```diff
@@ -12,6 +12,8 @@ function link {
 
 link /usr/lib/java/mariadb-java-client.jar $JBOSS_HOME/modules/system/layers/openshift/org/mariadb/main/mariadb-java-client.jar
 link /usr/share/java/postgresql-jdbc.jar $JBOSS_HOME/modules/system/layers/openshift/org/postgresql/main/postgresql-jdbc.jar
+link /usr/share/java/ongres-scram/common.jar $JBOSS_HOME/modules/system/layers/openshift/com/ongres/scram/common/main/ongres-scram-common.jar
```
Comment (davidesalerno) [inline jboss-kie-db-drivers/configure.sh:15]: Are we sure that adding the ongres-scram won't add any harm to the non-SSL connections?

Comment (spolti) [reply jboss-kie-db-drivers/configure.sh:15]: it shouldn't, but we can assure this later in the QE window.

**Review by jakubschwan — COMMENTED** [review]
lgtm

**Comment by Neustradamus** [comment]
Good job!

Linked to:
- https://github.com/scram-sasl/info/issues/1

**Pull request merged**